public/app/plugins/panel/alertlist/UnifiedalertList.test.tsx

```diff
@@ -159,18 +158,18 @@ const renderPanel = (options: Partial<UnifiedAlertListOptions> = defaultOptions)
 
   return render(
     <Provider store={store}>
-      <UnifiedAlertList {...props} />
+      <UnifiedAlertListPanel {...props} />
     </Provider>
   );
 };
 
 describe('UnifiedAlertList', () => {
+  jest.spyOn(contextSrv, 'hasPermission').mockReturnValue(true);
+
   it('subscribes to the dashboard refresh interval', async () => {
     jest.spyOn(defaultProps, 'replaceVariables').mockReturnValue('severity=critical');
 
-    await act(async () => {
-      renderPanel();
-    });
+    renderPanel();
 
     expect(dashboard.events.subscribe).toHaveBeenCalledTimes(1);
```

Comment:
(This would need to change to the below to get rid of the console.error:)

```js
await waitFor(() => expect(dashboard.events.subscribe).toHaveBeenCalledTimes(1));
```